File: js/tripleTrouble.js
// Create a function that will return a string that combines all of the letters of the three inputed strings in groups. Taking the first letter of all of the inputs and grouping them next to each other.

function tripleTrouble(one, two, three) {
  const newArr = []
  one.split('').map((el, index) => newArr.push(el + two[index] + three[index]))
  return newArr.join('')
}

console.log(tripleTrouble('Sea', 'urn', 'pms')) // returns "Supermans"

File: c#/TripleTrouble.cs
// return 1st letter of 3 inputs, then triple it

public class TripleTrouble
{
	public class Kata
	{
		public static string TripleTrouble(string one, string two, string three)
		{
			string stringBuilder = "";
			for (int i = 0; i < one.Length; i++) {
				stringBuilder += $"{one[i]}{two[i]}{three[i]}";
			}
			return stringBuilder;
		}
	}

	public static void Main()
	{
		Console.WriteLine(Kata.TripleTrouble("this","test","lock")); // expected: ttlheoiscstk
		Console.WriteLine(Kata.TripleTrouble("aa","bb","cc"));       // expected: abcabc
		Console.WriteLine(Kata.TripleTrouble("Bm","aa","tn"));       // expected: Batman
		Console.WriteLine(Kata.TripleTrouble("LLh","euo","xtr"));    // expected: LexLuthor
	}
}